packages/core/saved-objects/core-saved-objects-base-server-internal/src/serialization/serializer.ts

```diff
@@ -114,6 +115,7 @@ export class SavedObjectsSerializer implements ISavedObjectsSerializer {
       ...(originId && { originId }),
       attributes: _source[type],
       references: references || [],
+      ...(managed != null ? { managed } : {}),
```

Comment:
Is there a reason we're not defaulting to `false` here?

```ts
managed: managed ?? false,
```

Reply:
There are two options for populating the field: either through the serializer or as a migration. The thought process was that rather than bloat the serializer, we'd backfill `managed` during migration and leave it as `undefined` in existing docs.
If setting the default here won't affect performance, I guess it won't harm to set `managed: false` here too.